includes/Autopromote.php

```diff
@@ -28,45 +28,6 @@ public static function getAutopromoteGroups( User $user ) {
 	}
 
 	/**
-	 * Get the groups for the given user based on the given criteria.
```

Comment:
will we use Autopromote for anything after this PR is merged? have you considered removing everything related to autopromote?